Controller/Admin/OrderTabDetails.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use \OxidEsales\Eshop\Application\Controller\Admin\AdminDetailsController;
+use \OxidEsales\Eshop\Application\Model\Order;
+
+use \Wirecard\Oxid\Core\ResponseMapper;
+use \Wirecard\Oxid\Model\Transaction;
+
+/**
+ * Controls the view for the order details tab.
+ */
+class OrderTabDetails extends AdminDetailsController
```

Comment:
Why not extend from `TransactionTab`? It implements everything you did in `__construct()` and `render()`.

Reply:
created ListTab class as base class for TransactionTab and OrderTabDetails